packages/trader/README.md

```diff
@@ -66,6 +66,9 @@ npm run deploy:folder "br_my_test_folder"
 ## Miscellaneous
 - In Webstorm, right-click on `src`, hover over `Mark directory as`, and click `Resource root` to enable import alias resolution.  
 
+## Selenium tests
+Selenium test resides in every elements which have `id` attributes, with the value of `dt_[element_name]_[unique_name|id]_[element_type]]`. (e.g. `dt_settings_dark_button`)
```

Comment:
there is an extra `]` at the end.
can we also make the description clearer? `test resides` doesn't make much sense to me.